db/migrate/20221207102859_add_saved_locale_to_spree_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSavedLocaleToSpreeUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_users, :saved_locale, :string
```

Comment:
I would name the column simply `locale`, (or could be something like `selected_locale`) - the fact that it is being entered into the database means it is saved, so naming it "saved" is redundant ;)

Reply:
Yeah, that makes sense, I've changed it to selected_locale.